.gitignore

```diff
@@ -9,3 +9,6 @@ coverage.out
 
 # Bin directory
 bin/
+
+# Developer specific environment
+dev.env
```

Comment:
How do we generate this?

Reply:
Should have documented this, I made it manually. All I have in mine is `TS_AUTHKEY`. Just didn't want to keep pasting it in the shell